JASP-Desktop/po/jasp_nl.po

```diff
@@ -14957,7 +14956,8 @@ msgctxt "Form|"
 msgid ""
 "This analysis was created with an older version of JASP (or a dynamic module)"
 msgstr ""
-"Deze analyse is gemaakt met een oudere versie van JASP (of met een dynamische module)"
+"Deze analyse is gemaakt met een oudere versie van JASP (of met een "
+"dynamische module)"
```

Comment:
Why break the line in 2 ?

Reply:
That's what lupdate does. It is (of course) not a line break that is shoewn.

Reply:
lupdate does this (No real line braks of course)